README.md

```diff
@@ -103,6 +94,21 @@ http://localhost:3000
 A node server will continue running until you end the session.
 You can end the session by pressing `control-C`
 
+
+### Additional Configuration Options
+
+The [Official EOSDIS configurations](https://github.com/nasa-gibs/worldview-options-eosdis) are installed by default.   
+To use custom options, clone the template repository into an options directory in the root using the following git command.   
+Optionally, you can clone the official options and make changes as needed.
```

Comment:
I'd move this line in between the two `git clone` examples, since it refers to the latter.

Reply:
Sure, I can do that